tox.ini

```diff
@@ -38,19 +34,19 @@ extras =
     build_docs: docs
     all: all
 commands =
-    dev: pip install git+https://github.com/pypa/setuptools.git
+    devdeps: pip install git+https://github.com/pypa/setuptools.git
     pip freeze
     test: python -c 'import setuptools; print(setuptools.__version__)'
-    test: pytest --pyargs extension_helpers {toxinidir}/docs --cov extension_helpers --cov-config={toxinidir}/setup.cfg {posargs}
+    test: pytest --pyargs extension_helpers {toxinidir}/docs --cov extension_helpers --cov-config={toxinidir}/pyproject.toml {posargs}
     build_docs: sphinx-build -W -b html . _build/html
 
-[testenv:py39-downstream]
-changedir = test: .tmp/downstream
+[testenv:py{38,39,310,311,312}-downstream]
+changedir = .tmp/downstream
 commands =
     pip install setuptools setuptools_scm wheel cython numpy
-    pip install --no-build-isolation "astropy[test]@git+https://github.com/astropy/astropy.git@main"
+    pip install --no-build-isolation "astropy[test] @ git+https://github.com/astropy/astropy.git"
```

Comment:
You don't want to use astropy nightly wheel here?

Reply:
No because I want to test the build process, otherwise extension-helpers would be irrelevant